alerting/src/main/kotlin/org/opensearch/alerting/resthandler/RestGetRemoteIndexesAction.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.resthandler
+
+import org.apache.logging.log4j.LogManager
+import org.opensearch.alerting.AlertingPlugin
+import org.opensearch.alerting.action.GetRemoteIndexesAction
+import org.opensearch.alerting.action.GetRemoteIndexesRequest
+import org.opensearch.client.node.NodeClient
+import org.opensearch.core.common.Strings
+import org.opensearch.rest.BaseRestHandler
+import org.opensearch.rest.RestHandler
+import org.opensearch.rest.RestRequest
+import org.opensearch.rest.action.RestToXContentListener
+
+private val log = LogManager.getLogger(RestGetRemoteIndexesAction::class.java)
+
+class RestGetRemoteIndexesAction : BaseRestHandler() {
+    val ROUTE = "${AlertingPlugin.REMOTE_BASE_URI}/indexes"
```

Comment:
indices*

Reply:
I went with `indexes` because we had this issue a while back in which we were moving away from `indices`.
https://github.com/opensearch-project/index-management-dashboards-plugin/issues/637